.github/scripts/generate_pytorch_test_matrix.py

```diff
@@ -19,6 +20,13 @@ class Config(TypedDict):
     runner: str
 
 
+def get_disabled_issues() -> str:
+    pr_body = os.getenv('PR_BODY', '')
+    regex = '(?i)(Close(d|s)?|Resolve(d|s)?|Fix(ed|es)?) #([0-9]+)'
```

Comment:
can we have either comments or unit tests to demo some of the valid/invalid cases?

Reply:
Ah yes I can definitely add some comments

Reply:
FYI: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

Reply:
Yes, that's what I based it on. I added a comment with that link just now.